docs/5_1_scheduled_execution_math.adoc

```diff
@@ -1,19 +1,109 @@
 
 The Scheduled Execution interface has a different timing concept compared to FMI for Co-Simulation.
-This is required to cover <<clock>> ticks for <<periodic,aperiodic>> <<inputClock,`input clocks`>> which may tick at time instances that are not predictable in advance for the simulation algorithm.
+This is required to cover <<clock>> ticks for <<aperiodic-clocks,aperiodic>> <<inputClock,`input clocks`>> which may tick at time a instant that is unpredictable in advance for the simulation algorithm.
 Typically, hardware I/O or virtual ECU software events belong to this category.
 
-A simulation algorithm's call for computing a model partition will compute the results of the model partition defined by an <<inputClock>> for the current <<clock>> tick time latexmath:[t_i].
+A simulation algorithm's call for computing a model partition will compute the results of the model partition defined by an <<inputClock>> for the current <<clock>> tick time latexmath:[\mathbf{t}_i].
 
-The result values will be computed for the current <<clock>> tick time (activation time) latexmath:[t_i] from the assigned <<inputClock>> (which is known to the simulation algorithm).
-Refer to the <<clock>> time progress definition (<<CommunicationPointClocks>>) for <<periodic>> <<clock,`clocks`>>.
+The result values will be computed for the current <<clock>> tick time (activation time) latexmath:[\mathbf{t}_i] from the assigned <<inputClock>> (which is known to the simulation algorithm).
+Refer to the <<clock>> time progress definition for periodic <<clock,`clocks`>>.
 
-If required, the FMU can internally derive the <<clock>> interval latexmath:[\Delta T_i] based on the last <<clock>> tick time latexmath:[t_{i-1}] i.e. last activation time for this model partition.
+If required, the FMU can internally derive the <<clock>> interval latexmath:[\mathbf{T}_\mathit{interval, i}] based on the last <<clock>> tick time latexmath:[\mathbf{t}_\mathit{i-1}] i.e. last activation time for this model partition.
 
-A model partition can only be activated once per activation time point latexmath:[t_i].
+A model partition can only be activated once per activation time point latexmath:[\mathbf{t}_i].
 
-Model partitions that are associated to <<outputClock,`output clocks`>> will accordingly provide the result values of the model partition's variables for the current <<outputClock>> tick time latexmath:[t_i] of the active <<outputClock>>.
+Model partitions that are associated to <<outputClock,`output clocks`>> will accordingly provide the result values of the model partition's variables for the current <<outputClock>> tick time latexmath:[\mathbf{t}_i] of the active <<outputClock>>.
 The activation of such an <<outputClock>> is not controlled by the simulation algorithm but internally by the FMU.
 
-More details can be found in <<operation-on-clocks>>, specifically <<CommunicationPointClocks>>.
+More details can be found in <<clocks>>, specifically <<clocks>>.
```

Comment:
This looks like a typo? Two references to the same place?

Reply:
Fixed.